Header header
PedestrianPose[] poses
uint32 frameID